Build error in SIMDFeatures.h:25:17

## Enhancement

```
[build] In file included from /Users/zanmato/dev/tiflash/dbms/src/VectorSearch/SIMDFeatures.cpp:15:
[build] /Users/zanmato/dev/tiflash/dbms/src/VectorSearch/SIMDFeatures.h:25:17: error: no template named 'vector' in namespace 'std'
[build]    25 |     static std::vector<std::string> get();
[build]       |            ~~~~~^
[build] 1 error generated.
```

--- a/dbms/src/VectorSearch/SIMDFeatures.h
+++ b/dbms/src/VectorSearch/SIMDFeatures.h
@@ -15,6 +15,7 @@
 #pragma once
 
 #include <string>
+#include <vector>
 
 namespace DB::DM
 {

